troposphere/static/js/components/projects/resources/instance/details/sections/details/IpAddress.react.js

```diff
@@ -15,10 +15,17 @@ define(
       },
 
       render: function () {
+
+        var instance = this.props.instance;
+
+        var address = instance.get('ip_address');
+
+        if (!address || address.charAt(0) == "0") {
+            address = "N/A";
+        }
+
```

Comment:
Making this change in two places reminds me that we have components, like `IpAddress`, that you would _expect_ to be reusable - but have been copied/reimplemented. Don't worry about this now. But at some point we should review the existing components and see if we can eliminate duplication.